swarmzero/sdk_context.py

```diff
@@ -659,3 +671,82 @@ async def save_config_to_db(self):
 
         # Insert the data into the table
         await db_manager.insert_data("config", {"name": "default", "data": config_data, "create_date": datetime.now()})
+    
+    def get_log_level(self):
+        """
+        Get the log level from environment variable, then [logging].log_level from config,
+        then [log].level from config, or default to INFO.
+        
+        :return: The log level as a string (DEBUG, INFO, WARNING, ERROR, CRITICAL)
+        """
+        
+        env_log_level = os.environ.get("SWARMZERO_LOG_LEVEL")
+        if env_log_level:
+            return env_log_level.upper()
+        
+        logging_section = self.config.config.get("logging", {})
+        config_log_level = logging_section.get("log_level")
+        
+        if config_log_level:
+            return str(config_log_level).upper()
+
+        return "INFO"
+
+
+    def get_logging_config(self):
+        """
+        Get the logging configuration for the SDK context.
+
+        :return: A dictionary with logging configuration settings.
+        """
+        log_section = self.config.config.get("logging", {})
+
+        return {
+            "log_to_file":log_section.get("log_to_file", False),
```

Comment:
this would be redundant since we can just check if the log file path exists and is a valid path